scripts/.storybook/main.js

```diff
@@ -7,8 +7,6 @@ module.exports = {
     '@storybook/addon-essentials',
     '@storybook/addon-knobs',
     '@storybook/addon-a11y',
-    'storybook-prop-types-addon',
-    'storybook-addon-jsx',
```

Comment:
Can we remove these from `package.json` or do we still need them?

Reply:
Good catch, I thought I removed them, rebase, sorry!